src/Stryker.Core/Stryker.Core/ProjectComponents/IProjectComponent.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Stryker.Core.Mutants;
+using System.Collections.Generic;
+
+namespace Stryker.Core.ProjectComponents
+{
+    public interface IProjectComponent : IReadOnlyInputComponent
```

Comment:
I think this is the wrong way around. IProjectComponent cannot be a IReadOnlyInputComponent if it has mutable state.

Reply:
IProjectComponent expands upon IReadOnlyInputComponent by addding "IEnumerable<Mutant> Mutants { get; set; }" we can't put a mutable list in IReadOnlyInputComponent . What do you propose?

Reply:
In essence these two interfaces are not compatible right now. Let's talk about it later today.